alembic.ini

```diff
@@ -60,7 +60,7 @@ version_path_separator = os  # Use os.pathsep. Default configuration used for ne
 # are written from script.py.mako
 # output_encoding = utf-8
 
-sqlalchemy.url = 
+sqlalchemy.url = driver://user:pass@localhost/dbname
```

Comment:
It should be
sqlalchemy_url=

Nothing after that

Reply:
updating